single-table-inheritance/src/main/java/com/iluwatar/entity/TransportVehicle.java

```diff
@@ -0,0 +1,30 @@
+package com.iluwatar.entity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+/**
+ * An abstract class that extends the Vehicle class
+ * and provides properties for the Transport type of Vehicles.
+ *
+ * @see Vehicle
+ */
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+public abstract class TransportVehicle extends Vehicle {
+
+  private int loadCapacity;
+
+  public TransportVehicle(String manufacturer, String model, int loadCapacity) {
+    super(manufacturer, model);
+    this.loadCapacity = loadCapacity;
+  }
+
+  @Override
+  public String toString() {
+    return super.toString();
+  }
```

Comment:
seems unnecessary override

Reply:
Yes, I think I missed it. Willl remove it